docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-oncall.md

```diff
@@ -1,4 +1,6 @@
 ---
+aliases:
+- ../../../alerting-rules/manage-contact-points/integrations/configure-oncall/ # /docs/grafana/latest/alerting/alerting-rules/manage-contact-points/integrations/configure-oncall/
```

Comment:
You also need to remove the aliases section in  lines 11 and 12: https://github.com/grafana/grafana/pull/83372/files#diff-a39da9a6af6c80a585251b83bc4dc2dc2d36cfc7b6e9d88ce3bdf8a47805b025R11-R12

```suggestion
  - ../../../alerting-rules/manage-contact-points/configure-oncall/ # /docs/grafana/latest/alerting/alerting-rules/manage-contact-points/configure-oncall/
  - ../../../alerting-rules/manage-contact-points/integrations/configure-oncall/ # /docs/grafana/latest/alerting/alerting-rules/manage-contact-points/integrations/configure-oncall/
```